src/components/Skills.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Container} from "react-bootstrap";
+import pythonlogo from "../assets/img/python-logo.png"
+import reactlogo from "../assets/img/react-logo.png"
+import csslogo from "../assets/img/css-logo.png"
+import htmllogo from "../assets/img/html-logo.png"
+import clogo from "../assets/img/c++-logo.png"
+import javascriptlogo from "../assets/img/javascript-logo.png"
+import awslogo from "../assets/img/aws-logo.png"
+import nodejslogo from "../assets/img/nodejs-logo.png"
+import expressjslogo from "../assets/img/express-logo.png"
+import mongodblogo from "../assets/img/mongodb-logo.png"
+
+export const Skills = () => {
+
+    return (
+        <section className="skills">
+            <Container className="container-1">
+                <h1>TECH STACK</h1>
+            </Container>
+            <Container className="skills-container">
+                <div>
```

Comment:
We can create an array of object with property for the name/title and the image
```
const techStacks = [{title: AWS, img: awslogo}, ...] 
```
then use Array.map, to render the tech stacks